participants/karthick.json

```
@@ -0,0 +1,53 @@
+// vim: ft=jsonc
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Karthick",
+		"familyName": "Shanmugam",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "karthick-shanmugam0689",
+	// company name (optional)
+	"company": "Sixt SE",
+	// either both days or at least one day has to be set to true.
+	// If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+	"when": {
+		// June 27th, 2025
+		"friday": true,
```

It would be best to split it up into two PRs then - remove Friday attendance here and we can merge, adding another PR for setting it to `true` again to put you on the waiting list for Friday

```suggestion
		"friday": false,
```